modules/csg/csg.cpp

```diff
@@ -33,167 +33,13 @@
-	Vector3 q = s.cross(edge1);
-	real_t v = f * p_dir.dot(q);
-	if (v < 0.0 - p_tolerance || u + v > 1.0 + p_tolerance) {
-		return false;
-	}
-
-	// Ray intersects triangle.
-	// Calculate distance.
-	real_t t = f * edge2.dot(q);
-	// Confirm triangle is in front of ray.
-	if (t >= p_tolerance) {
-		r_intersection_point = p_from + p_dir * t;
-		return true;
-	} else {
-		return false;
-	}
-}
-
-inline bool is_point_in_triangle(const Vector3 &p_point, const Vector3 p_vertices[3], int p_shifted = 0) {
-	real_t det = p_vertices[0].dot(p_vertices[1].cross(p_vertices[2]));
-
-	// If determinant is, zero try shift the triangle and the point.
-	if (Math::is_zero_approx(det)) {
-		if (p_shifted > 2) {
-			// Triangle appears degenerate, so ignore it.
-			return false;
-		}
-		Vector3 shift_by;
-		shift_by[p_shifted] = 1;
-		Vector3 shifted_point = p_point + shift_by;
-		Vector3 shifted_vertices[3] = { p_vertices[0] + shift_by, p_vertices[1] + shift_by, p_vertices[2] + shift_by };
-		return is_point_in_triangle(shifted_point, shifted_vertices, p_shifted + 1);
-	}
-
-	// Find the barycentric coordinates of the point with respect to the vertices.
-	real_t lambda[3];
-	lambda[0] = p_vertices[1].cross(p_vertices[2]).dot(p_point) / det;
-	lambda[1] = p_vertices[2].cross(p_vertices[0]).dot(p_point) / det;
-	lambda[2] = p_vertices[0].cross(p_vertices[1]).dot(p_point) / det;
-
-	// Point is in the plane if all lambdas sum to 1.
-	if (!Math::is_equal_approx(lambda[0] + lambda[1] + lambda[2], 1)) {
-		return false;
-	}
-
-	// Point is inside the triangle if all lambdas are positive.
-	if (lambda[0] < 0 || lambda[1] < 0 || lambda[2] < 0) {
-		return false;
-	}
-
-	return true;
-}
-
-inline static bool is_triangle_degenerate(const Vector2 p_vertices[3], real_t p_vertex_snap2) {
-	real_t det = p_vertices[0].x * p_vertices[1].y - p_vertices[0].x * p_vertices[2].y +
-			p_vertices[0].y * p_vertices[2].x - p_vertices[0].y * p_vertices[1].x +
-			p_vertices[1].x * p_vertices[2].y - p_vertices[1].y * p_vertices[2].x;
-
-	return det < p_vertex_snap2;
-}
-
-inline static bool are_segments_parallel(const Vector2 p_segment1_points[2], const Vector2 p_segment2_points[2], float p_vertex_snap2) {
-	Vector2 segment1 = p_segment1_points[1] - p_segment1_points[0];
-	Vector2 segment2 = p_segment2_points[1] - p_segment2_points[0];
-	real_t segment1_length2 = segment1.dot(segment1);
-	real_t segment2_length2 = segment2.dot(segment2);
-	real_t segment_onto_segment = segment2.dot(segment1);
-
-	if (segment1_length2 < p_vertex_snap2 || segment2_length2 < p_vertex_snap2) {
-		return true;
-	}
-
-	real_t max_separation2;
-	if (segment1_length2 > segment2_length2) {
-		max_separation2 = segment2_length2 - segment_onto_segment * segment_onto_segment / segment1_length2;
-	} else {
-		max_separation2 = segment1_length2 - segment_onto_segment * segment_onto_segment / segment2_length2;
-	}
-
-	return max_separation2 < p_vertex_snap2;
-}
+#include "thirdparty/manifold/include/manifold/manifold.h"
```

Comment:
Adding `thirdparty_dir + "include"` to the `CPPPATH` is so that you can do this:
```suggestion
#include <manifold/manifold.h>
```

May or may not be relevant for manifold in the future, but that's also the syntax which would work if we supported building against a system packaged manifold library on Linux. (Again here that's highly theoretical, I don't suggest working on adding support for that - I don't see any Linux distro packaging it.)